src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -115,31 +112,6 @@ describe("Decimal", () => {
     return press({ clipboardData }, where, "paste");
   };
 
-  let translations;
-
-  testStyledSystemMargin(
```

Comment:
We still need this test to cover that the styled-system margin props are applied to Decimal.